config/locales/en.yml

```diff
@@ -36,3 +36,10 @@ en:
       sent_to_cds: "Sent to CDS"
       cds_import_error: "CDS import error"
       already_in_cds: "Already in CDS"
+
+  create_measures:
+    errors:
+      blank_workbasket_name: "Workbasket name can not be blank."
+      blank_commodity_and_additional_codes: "Goods commodity codes: can be empty but only if one or more Meursing codes are entered in the Additional codes field."
+      commodity_codes_exclusions: "Exceptions: if one or more codes are entered here, Goods commodity codes field must not be empty; each exception code must be a child of one of the goods commodity codes above."
+      commodity_codes_invalid: "There are no any valid code specified in 'Goods commodity codes' or 'Additional codes' inputs! Please, double check params"
```

Comment:
You need to enter valid codes in the 'Goods commodity codes' or 'Additional codes' fields.